content/docs/faq/cc-bcc-with-rest-api.md

```diff
@@ -124,3 +124,75 @@ Finally, here's an example with both CC and BCC recipients.  You set header_to
   }
 }
 ```
+
+### Using CC and BCC with Stored Templates
+
+Header values must be set directly within the stored template, so manually adding CC and BCC will not be honored when a stored template is used. To use CC and BCC with a stored template, the template must be created through the API.
+
+One method is to use substitution data for your templates that you want to have a CC line. For example, you could create a template like this:
+
+```
+{
+        "id": "cc_template_3",
+        "name": "cc_template_3",
+        "description": "",
+        "published": true,
+        "options": {
+            "click_tracking": true,
+            "open_tracking": false
+        },
+        "shared_with_subaccounts": false,
+        "last_update_time": "2018-12-14T17:15:12+00:00",
+        "last_use": "2018-12-14T17:15:49+00:00",
+        "has_published": true,
+        "has_draft": false,
+        "content": {
+            "from": "you@fromyou.com",
+            "headers": {
+                "CC": "{{ cc_data or '' }}"
+            },
+            "subject": "CC 2",
+            "text": " cc email"
+        }
+}
+```
+
+Then, for the transmission, set the substitution data for the CC:
+
+```
+{
+  "options" : {
+    "open_tracking" : true,
+    "click_tracking" : true,
+    "conversion_tracking" : false,
+    "transactional": true
+  },
+  "campaign_id" : "mycampaign-msys",
+  "return_path" : "you@bounce.fromyou.com",
+  "header_from" : "Single Recipient",
+  "metadata" : {
+    "key": "value"
+  },
+  "recipients" : [
+    {
+      "address" : {
+        "email": "to@thisperson.com"
+      },
+      "tags" : [
+        "greeting", "prehistoric", "fred", "flintstone"
+        ],
+      "header_to" : "Wilma Flintstone",
+      "metadata" : {
+        "place" : "Bedrock",
+        "host" : "google.com"
+      }
+    }
+  ],
+  "substitution_data": {
+  	"cc_data": "cc@thatperson.com"
+  },
+  "content" : {
+   	"template_id": "cc_template_3"
+  }
+}
+```
```

Comment:
Let's also format this block and enable syntax highlighting:
````suggestion
```json
{
  "options": {
    "open_tracking": true,
    "click_tracking": true,
    "conversion_tracking": false,
    "transactional": true
  },
  "campaign_id": "mycampaign-msys",
  "return_path": "you@bounce.fromyou.com",
  "header_from": "Single Recipient",
  "metadata": {
    "key": "value"
  },
  "recipients": [
    {
      "address": {
        "email": "to@thisperson.com"
      },
      "tags": [
        "greeting",
        "prehistoric",
        "fred",
        "flintstone"
      ],
      "header_to": "Wilma Flintstone",
      "metadata": {
        "place": "Bedrock",
        "host": "google.com"
      }
    }
  ],
  "substitution_data": {
    "cc_data": "cc@thatperson.com"
  },
  "content": {
    "template_id": "cc_template_3"
  }
}
```
````